src/components/paintings-page/collections-drop-down/collections-drop-down.module.scss

```diff
@@ -0,0 +1,102 @@
+@use '@variables' as var;
```

Comment:
All this styling looks great, but haven't you already styled most of this component in the original dropdown file?